datacite-findable-doi-handler/src/main/java/no/unit/nva/datacite/handlers/LandingPageUtil.java

```diff
@@ -0,0 +1,51 @@
+package no.unit.nva.datacite.handlers;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+public final class LandingPageUtil {
```

Comment:
TODO: Add class purpose, unsure where the ownership of this one should be. Any idea?

Reply:
Fixed in https://github.com/BIBSYSDEV/nva-doi-registrar-client/pull/28/commits/31214e01b4eb6682f516f759b2a131cfffa15047 . 

This should be moved into a separate module under nva-publication-api when the second user needs to generate the landing page URIs for publications.